nuxt.config.ts

```diff
@@ -40,6 +40,9 @@ export default defineNuxtConfig({
     documentDriven: true,
     highlight: {
       theme: 'one-dark-pro'
+    },
+    markdown: {
+      rehypePlugins: ['rehype-external-links']
```

Comment:
Configure target option for external links: https://github.com/rehypejs/rehype-external-links#optionstarget

```suggestion
      rehypePlugins: {
        'rehype-external-links': {
          target: '_blank'
        }
      }
```